README.rst

```diff
@@ -19,28 +19,43 @@ OpenWISP 2 monitoring module (Work in progress).
 
 ------------
 
-Install Depdendencies
+Install Dependencies
 ---------------------
```

Comment:
The length of this line must be equal to the text in the previous line.

Reply:
yeah there's one extra dash `-`, some ReStructuredText parsers don't like that